### ISSUE TITLE
Chinese language news channels suggestions (Youtube)

### ISSUE THREAD

**Issue by tyetye** [Fleker/CumulusTV#98]
I saw your Sky News YouTube channel comment saying it should be easy to add YouTube support. If so, here are a few mostly Taiwanese channels that have stable URLs:
CTI: https://www.youtube.com/watch?v=hgIfZz8STLk
Formosa: https://www.youtube.com/watch?v=XxJKnDLYZz4
UDN: https://www.youtube.com/watch?v=SwrKzkRUlaw

Of these, UDN is probably the least gossipy and focuses on international, business, and domestic affairs. If you are interested, there are a few others I can suggest but I'd have to research because those sometimes change the URL of their live streams.

I hope adding any live YouTube streams can be easy in the future. It looks like partial support is in the current Play Store build?

**Comment by Fleker** [comment]
Yes there is currently support for loading any website as a live channel.
YouTube should be easy by using the embed URL.

**Comment by nfgb** [comment]
Hi, is the support for loading any website as a live channel still working?
I've tried lots of websites and all I get is a black screen even the example site: http://time.is.
I assume you only need to put the url of the website on the media url field when you create the channel.
